suite-native/module-accounts-import/src/components/AccountImportHeader.tsx

```diff
@@ -10,19 +23,51 @@ const accountImportHeaderStyle = prepareNativeStyle(_ => ({
     width: '100%',
 }));
 
+const closeButtonSizeStyle = prepareNativeStyle(_ => ({
+    width: 48,
+    height: 48,
+}));
+
+type NavigationProp = StackToTabCompositeProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
 export const AccountImportHeader = ({ activeStep }: AccountImportHeaderProps) => {
     const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProp>();
+    const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
+
+    const handleCloseOnboarding = () =>
+        navigation.navigate(RootStackRoutes.AppTabs, {
+            screen: AppTabsRoutes.HomeStack,
+            params: {
+                screen: HomeStackRoutes.Home,
+            },
+        });
+
     return (
         <Box
             flexDirection="row"
-            justifyContent="center"
+            justifyContent={isOnboardingFinished ? 'space-between' : 'center'}
             alignItems="center"
             marginBottom="small"
             style={applyStyle(accountImportHeaderStyle)}
         >
-            <Box>
-                <StepsProgressBar activeStep={activeStep} numberOfSteps={3} />
-            </Box>
+            {isOnboardingFinished && <Box style={applyStyle(closeButtonSizeStyle)} />}
```

Comment:
I don't like this way of styling where we placing empty boxes to make things centered. Please try to find better way, I pretty sure this could be done in flex layout using some props like `alignSelf: flex-end` or something similar.

Reply:
Yeah it could be. But unfortunately when I do that, the progress bar is not centered completely. Even if this first item would have different width, the middle item wouldn't be centered perfectly. I was looking for better solution fair amount of time and couldn't find a better way to do this. 

`alignSelf` won't work because it's row so this won't be possible. Maybe @dahaca might have some cool hack up his sleeve?

Reply:
I would probably go with this:

![Screenshot 2022-11-02 at 16 52 35](https://user-images.githubusercontent.com/5837757/199537534-c6c01077-e938-440b-bef9-4f8c642da2d6.png)

It's not ideal either because you need to specify height of container, but at least you don't need to add components to act just as placeholder. Also this placeholder solution will even break if we want to add different sized button to left side for example.

Also please create component for this header part, it's repeated on every screen in account imports.

Reply:
Styles fixed. I used position absolute as suggested. Even though I wanted to avoid this but I guess it's better than the empty box.